ports/ogre-next/portfile.cmake

```
@@ -59,63 +58,31 @@ vcpkg_configure_cmake(
         -DOGRE_BUILD_RENDERSYSTEM_GL3PLUS=ON
         -DOGRE_BUILD_RENDERSYSTEM_GLES=OFF
         -DOGRE_BUILD_RENDERSYSTEM_GLES2=OFF
-# Optional stuff
-        ${FEATURE_OPTIONS}
-# vcpkg specific stuff
         -DOGRE_CMAKE_DIR=share/ogre-next
 )
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup()
 
-vcpkg_install_cmake()
-vcpkg_fixup_cmake_targets()
-
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
-
-file(GLOB REL_CFGS ${CURRENT_PACKAGES_DIR}/bin/*.cfg)
+file(GLOB REL_CFGS "${CURRENT_PACKAGES_DIR}/bin/*.cfg")
 if(REL_CFGS)
-  file(COPY ${REL_CFGS} DESTINATION ${CURRENT_PACKAGES_DIR}/lib)
-  file(REMOVE ${REL_CFGS})
+  file(COPY "${REL_CFGS}" DESTINATION "${CURRENT_PACKAGES_DIR}/lib")
+  file(REMOVE "${REL_CFGS}")
 endif()
 
-file(GLOB DBG_CFGS ${CURRENT_PACKAGES_DIR}/debug/bin/*.cfg)
+file(GLOB DBG_CFGS "${CURRENT_PACKAGES_DIR}/debug/bin/*.cfg")
 if(DBG_CFGS)
-  file(COPY ${DBG_CFGS} DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib)
-  file(REMOVE ${DBG_CFGS})
+  file(COPY "${DBG_CFGS}" DESTINATION "${CURRENT_PACKAGES_DIR}/debug/lib")
+  file(REMOVE "${DBG_CFGS}")
```

```suggestion
  file(COPY ${DBG_CFGS} DESTINATION "${CURRENT_PACKAGES_DIR}/debug/lib")
  file(REMOVE ${DBG_CFGS})
```
`DBG_CFGS` is a list, not a path.